opensearch/src/root/bulk.rs

```diff
@@ -245,6 +250,22 @@ impl<B> BulkCreateOperation<B> {
         }
     }
 
+    /// Creates a new instance of [BulkCreateOperation] without explicit id
+    pub fn new_no_id(source: B) -> Self {
```

Comment:
I'd prefer `without_id` over `no_id` for comprehensibility.
```suggestion
    /// Creates a new instance of [BulkCreateOperation] without an explicit id
    pub fn new_without_id(source: B) -> Self {
```